services/src/contexts/mod.rs

```diff
@@ -89,18 +89,13 @@ impl<D> ExecutionContext for ExecutionContextImpl<D>
 where
     D: DataSetDb
         + MetaDataProvider<MockDataSetDataSourceLoadingInfo, VectorResultDescriptor>
-        + MetaDataProvider<OgrSourceDataset, VectorResultDescriptor>,
+        + MetaDataProvider<OgrSourceDataset, VectorResultDescriptor>
+        + MetaDataProvider<GdalLoadingInfo, RasterResultDescriptor>,
```

Comment:
just a thought: we could also think about implementing the actual MEtaDataProviders as separate structs which get a database object. then we can composition them together (or wrap them in an enum or something).

Reply:
it's possible but then each provider needs e.g. a separate db pool reference etc. even though they all belong to the same database. Also makes the in memory implementation harder imho.